src/NuGet.Clients/PackageManagement.UI/Xamls/PackageManagerControl.xaml.cs

```diff
@@ -840,5 +855,84 @@ private void SuppressDisclaimerChecked(object sender, RoutedEventArgs e)
             _legalDisclaimer.Visibility = Visibility.Collapsed;
             RegistrySettingUtility.SetBooleanSetting(Constants.SuppressUIDisclaimerRegistryName, true);
         }
+
+        public void ExecuteAction(UserAction action, Action<NuGetUI> setOptions)
```

Comment:
Extract action execution into this new method since now it's called from multiple places:
- The button in the right pane
- The install, update or uninstall button in the left package list when in project package manager.